publish-layers.sh

```diff
@@ -2,6 +2,7 @@
 
 NODEJS610_DIST=dist/nodejs610.zip
```

Comment:
Do we want to remove the node 6 layer here?

Reply:
Created an issue for that: https://github.com/iopipe/iopipe-js/issues/50

AWS is still allowing layers to be published for `nodejs6.10`, though.

Reply:
Ah, a good reason to keep it around, then!